src/util/cron.c

```diff
@@ -13,10 +13,11 @@
 //------------------------------------------------------------------------------
 
 typedef enum {
-	TASK_PENDING   = 0,  // task is waiting for executing
-	TASK_EXECUTING = 1,  // task is being executed
-	TASK_COMPLETED = 2,  // task run to completion
-	TASK_ABORT     = 3   // task is aborted
+	TASK_INVALID   = 0,  // invalid task
```

Comment:
I'm not convinced we need `TASK_INVALID`, `0` is a result of an allocator free which decides to zero out the memory.